galleon-pack/galleon-feature-pack/pom.xml

```diff
@@ -72,6 +73,28 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-resources-plugin</artifactId>
                 <executions>
+                    <execution>
+                        <id>copy-ee-galleon-shared-feature-pack-resources</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <escapeString>\</escapeString>
+                            <outputDirectory>${basedir}/target/resources/packages/</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <targetPath>${basedir}/target/resources/packages/docs.licenses</targetPath>
+                                    <directory>${ee.galleon.shared.resources.directory}/packages/docs.licenses</directory>
+                                </resource>
+                                <resource>
+                                    <targetPath>${basedir}/target/resources/packages/docs.licenses.merge</targetPath>
+                                    <directory>${ee.galleon.shared.resources.directory}/packages/docs.licenses.merge</directory>
+                                </resource>
+                            </resources>
+                            <overwrite>true</overwrite>
+                        </configuration>
+                    </execution>
```

Comment:
This is ok for now but we'll definitely refactor how this works at some point.

But this is fine; the whole business of using the maven-resources-plugin to compose feature packs needs to be replaced; the way you need to deal with this use case here is more evidence of that.